spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/web/format/WebConversionServiceTests.java

```diff
@@ -79,6 +79,17 @@ void isoTimeFormat() {
 		assertThat(conversionService.convert(time, String.class))
 				.isEqualTo(DateTimeFormatter.ISO_LOCAL_TIME.format(time));
 	}
+	
+		@Test
+	void isoOffsetTimeFormat() {
+		WebConversionService conversionService = new WebConversionService(new DateTimeFormatters().timeFormat("isooffset"));
+		OffsetDateTime offsetdate =OffsetDateTime.of(LocalDate.of(2020, 4, 26), LocalTime.of(12, 45, 23), ZoneOffset.ofHoursMinutes(1, 30)); 
```

Comment:
You're really close. Thank you. This needs to be an `OffsetTime` rather than an `OffsetDateTime`:

```java
OffsetTime offsetdate = OffsetTime.of(LocalTime.of(12, 45, 23), ZoneOffset.ofHoursMinutes(1, 30));
```

With this change in place I think your test will pass.

Reply:
Thank you I have committed the code after fix. CI build still seems to be failing rest all checks are passing fine. Could you please assign any other issue I could take up please?